package Exceptions_and_Errors;

public class ExceptionExample {
    public static void main(String[] args) {
        try {
            int[] numbers = {1, 2, 3};
            System.out.println(numbers[5]); // This will throw an ArrayIndexOutOfBoundsException
        } catch (ArrayIndexOutOfBoundsException e) {
            System.out.println("Array index is out of bounds! " + e.getMessage());
        } finally {
            System.out.println("This block always executes.");
        }
    }
}

